README.md

```diff
@@ -161,6 +161,15 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+PAM logins
+----------
+
+Starting with v2.0.0, the python iRODS client is able to authenticate under PAM using the same file-based client environment as the
+iCommands.
+
+Caveat for iRODS 4.3+: when upgrading from 4.2, the "irods_authentication_scheme" setting must be changed from "pam" to "pam_password" in
```

Comment:
the language here shifts to versions of the iRODS server... but we can tweak it later.

Reply:
Ok ... Not quite sure how I'd fix that one at present.

Reply:
> the language here shifts to versions of the iRODS server... but we can tweak it later.

New issue then?

Reply:
yes please - make a new issue and we'll get there...

Reply:
For complete clarity: Issue created for this is #546